test/rules/no-shadowed-variable/ignore-underscore/test.ts.lint

```diff
@@ -0,0 +1,5 @@
+import _ = require('lodash');
+
+function foo(_) {
```

Comment:
What about multiple unused parameters? The rule should also apply to names like `_a`, `_match`, etc., right?

```ts
function foo(_a, _b) { }
```

TypeScript doesn't allow duplicate identifiers of the same name unfortunately.

Reply:
Thanks, forgot about this case!